src/components/modals/ant-management/UpgradeAntModal/UpgradeAntModal.tsx

```diff
@@ -98,6 +100,11 @@ function UpgradeAntModal({
           processId: antId,
           luaCodeTxId: ANT_LUA_ID,
           signer,
+          ao: connect({
```

Comment:
Is it a problem to cache ao outside of this loop and share or cache globally and share?

Also seeing CU_URL, MU_URL, and SCHEDULER makes it look like global constants. Would rename those to something like cuUrl, muUrl, and scheduler.

Reply:
Nevermind about the casing for CU_URL, etc., I see that's a part of aoconnect. :(

Reply:
But still curious about caching ao :)

Reply:
They are global constants.

Reply:
They're not, they're key names to the object passed into connect. We can't change that. i'm more concerned about caching ao vs. having to recreate each time.